### PR TITLE
Add commonLabels to helm chart

### PR DESCRIPTION
**Pull request by alaturqua**
## Description
This PR adds commonLabels to all resources. For some kubernetes instances, it is required to have custom labels.

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
(x) Release notes are required, with the following suggested text:

```markdown
Add commonLabels to helm chart 
```

### CODE REVIEW

File: helm/trino-gateway/values.yaml [hunk 1]
```diff
@@ -110,6 +111,11 @@ tolerations: []
 
 affinity: {}
 
+commonLabels: {}
+  # io.example/cost-center: "000000"
+  # io.example/team: "trino"
```
Comment (mosabua) [inline helm/trino-gateway/values.yaml:116]: Do the labels have to have a namespace or can it also be

example: "some value"

**Review by willmostly — APPROVED** [review]
This looks good. @alaturqua could you squash your commits? Please capitalize your commit message as per the guidance here: https://cbea.ms/git-commit/

File: helm/trino-gateway/values.yaml (hunk 1 above)
Comment (mosabua) [reply helm/trino-gateway/values.yaml:116]: If yes .. we should add such a simpler example. Also

File: helm/trino-gateway/values.yaml [hunk 2]
```diff
@@ -110,6 +111,11 @@ tolerations: []
 
 affinity: {}
 
+commonLabels: {}
```
Comment (mosabua) [inline helm/trino-gateway/values.yaml:114]: Add docs like in https://github.com/trinodb/charts/blob/ac81a3cc14b766d3c2b2ed70e85863f67dc36a2a/charts/trino/values.yaml#L799

That way we can produce docs later like in the Trino chart..

**Review by mosabua — CHANGES_REQUESTED** [review]
Looks good to me, just some needs and probably want a look from @willmostly

File: helm/trino-gateway/values.yaml (hunk 1 above)
Comment (alaturqua) [reply helm/trino-gateway/values.yaml:116]: It does need to have namespace. That was just the way we were using commonLabels. I can change them and add comment for docs.

File: helm/trino-gateway/values.yaml (hunk 2 above)
Comment (alaturqua) [reply helm/trino-gateway/values.yaml:115]: added

**Comment by alaturqua** [comment]
@willmostly 
Squashed my commits and added docs. Please review again.

**Review by mosabua — APPROVED** [review]
Looks good now. Thanks @alaturqua

**Pull request merged**